src/Pact/Compile.hs

```diff
@@ -413,7 +413,10 @@ defpact = do
 moduleForm :: Compile (Term Name)
 moduleForm = do
   modName' <- _atomAtom <$> userAtom
-  keyset <- str
+  gov <- Governance <$>
+    (((Left . KeySetName) <$> str) <|>
+     (userAtom >>= \AtomExp{..} ->
+         return $ Right $ TVar (Name _atomAtom _atomInfo) _atomInfo))
```

Comment:
Performance rule of thumb -- (almost) always use `return $!` instead of `return $` to avoid thunk creation. The second `$` could/should be `$!` also

Reply:
This would seem to better be part of a larger review -- there are a lot of `return (foo)` in this file alone. We need to get a general Pact performance improvement effort going.